glossary.dd

```diff
@@ -81,6 +81,14 @@ $(DL
 	maximize the portability of code.
 	)
 
+	$(DT $(LNAME2 lvalue, lvalue))
+	$(DD An lvalue is an abstract term referring to a value contained within a
+	variable (including elements of arrays and associative arrays).
+	Characteristics of lvalues are they have a memory address, they can be
+	assigned new values and can be passed to (or be returned from) functions by
+	reference. An lvalue is the counterpart to an rvalue.
+	)
+
```

Comment:
> An lvalue is an abstract term

"Lvalue" is a (natural language) term, but *an* lvalue is an expression (in the programming language sense), no?

> referring to a value contained within a variable

Lvalues don't have to be variables. For example, `*(new int)` is an lvalue.

> Characteristics of lvalues are they have a memory address,

Plural "lvalues" conflicts with singular "memory address". So, "**an** lvalue" or "memory address**es**".
Also, "**that** they have"? I'm not sure.

> they can be assigned new values

They may be immutable, i.e. non-assignable.

> An lvalue is the counterpart to an rvalue.

This is about "lvalue" the term again. And maybe make "rvalue" the subject:
"The counterpart to the term lvalue is the term rvalue."
or
"The counterpart to lvalues are rvalues."
or something better I can't think of.
Also make "rvalues" a link.

Much of this applies to the rvalue entry as well.